src/proj.rs

```diff
@@ -916,6 +916,85 @@ impl Proj {
         self.array_general(points, Transformation::Projection, inverse)
     }
 
+    /// Transform boundary densifying the edges to account for nonlinear transformations along
+    /// these edges and extracting the outermost bounds.
+    ///
+    /// Input and output CRS may be specified in two ways:
+    /// 1. Using the PROJ `pipeline` operator. This method makes use of the [`pipeline`](http://proj4.org/operations/pipeline.html)
+    /// functionality available since `PROJ` 5.
+    /// This has the advantage of being able to chain an arbitrary combination of projection, conversion,
+    /// and transformation steps, allowing for extremely complex operations ([`new`](#method.new))
+    /// 2. Using EPSG codes or `PROJ` strings to define input and output CRS ([`new_known_crs`](#method.new_known_crs))
+    ///
+    /// The following example converts from NAD83 US Survey Feet (EPSG 2230) to NAD83 Metres (EPSG 26946)
+    ///
+    /// ```rust
+    /// # use approx::assert_relative_eq;
+    /// extern crate proj;
+    /// use proj::{Proj, Coord};
+    ///
+    /// let from = "EPSG:2230";
+    /// let to = "EPSG:26946";
+    /// let ft_to_m = Proj::new_known_crs(&from, &to, None).unwrap();
+    /// let result = ft_to_m
+    ///     .transform_bounds(4760096.421921, 3744293.729449, 4760196.421921, 3744393.729449, 21)
+    ///     .unwrap();
+    /// assert_relative_eq!(result[0] as f64, 1450880.29, epsilon=1e-2);
+    /// assert_relative_eq!(result[1] as f64, 1141263.01, epsilon=1e-2);
+    /// assert_relative_eq!(result[2] as f64, 1450910.77, epsilon=1e-2);
+    /// assert_relative_eq!(result[3] as f64, 1141293.49, epsilon=1e-2);
+    /// ```
+    ///
+    /// # Safety
+    /// This method contains unsafe code.
+    pub fn transform_bounds<F>(
+        &self,
+        left: F,
+        bottom: F,
+        right: F,
+        top: F,
+        densify_pts: i32,
```

Comment:
A doc about densify pts would be nice.

Reply:
I added a sentence taken from the pyproj docs